workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Writers/IWriter.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Improbable.Gdk.Core.GameObjectRepresentation
+{
+    public interface IWriter<TSpatialComponentData, TComponentUpdate>
+        : IReader<TSpatialComponentData, TComponentUpdate>
+        where TSpatialComponentData : ISpatialComponentData
+        where TComponentUpdate : ISpatialComponentUpdate
+    {
+        void Send(TComponentUpdate update);
+        void SendAuthorityLossImminentAcknowledgement();
+        void CopyAndSend(TComponentUpdate update);
```

Comment:
Do we need these semantics?

Reply:
`CopyAndSend`

Reply:
They're in the writers in the SDKs, I am happy to remove them if we don't want to support that :)

Reply:
I think leave out for now and add back in if we feel its necessary? 

This is the GDK monobehaviour support - not the SDK semantics shim layer for porting, right?

Reply:
I think more of the former yes, removing

Reply:
The codegened update type is also always a struct which is passed by value so the whole point of copying the update becomes moot.

Reply:
Unless you have ref types in the struct

Reply:
Yes, it still needs to be impl'd for non-blittable components and/or fields

Reply:
But even as a struct they can contain non-blittable fields and may need to be deep-copied if we want to support that